content/numpy/concepts/math-methods/terms/abs/abs.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: '.abs()'
+Description: 'Calculates the absolute value of a number or each element in an array.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'NumPy'
+  - 'Arrays'
+  - 'Functions'
+  - 'Math'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+The **`.abs()`** function calculates the absolute value of a number or the absolute value of all of the elements in an array. The absolute value of a number is the number but without a sign, or the number's distance from zero. `.abs()` can also compute the absolute value of complex numbers in the form of $a + bi$ by computing $\sqrt{a^2 + b^2}$. Additionally, `.absolute()` can be used instead of `.abs()`.
+
+## Syntax
+
+```psuedo
+`numpy.absolute(array, out=None, where=True)`
+```
+
+- `array`: The number or an array containing the elements to which the absolute value will be computed.
+- `out`: The location where the result of the absolute value will be stored. If no value is provided, the default value of `None` is used and a new array is returned.
+- `where`: An array of boolean values used to determine which elements of the array the absolute value function should be applied. If the condition is `True` at a given position, the absolute value is computed for that element, whereas if the condition is `False`, the original value is retained. If no value is provided, the default value is `True`, meaning that the absolute value is computed for all of the elements.
```

Comment:
if possible, Please concise this.

Reply:
Tried to make it more concise and reorganized it in the latest commit.